server/views/pages/unlock-list/planned-events.njk

```diff
@@ -163,87 +152,91 @@
+                            text: item.locationSubGroup + ' ' + item.cellLocation,
+                            classes: 'govuk-table_cell'
+                        },
+                        {
+                            attributes: { id: 'activity-' + loop.index },
+                            html: showUnlockEvents(item.events),
+                            classes: 'govuk-table__cell'
+                        },
+                        {
+                            html: '<span class="print-checkbox">R</span>',
+                            classes: 'govuk-table_cell govuk-!-padding-top-2 govuk-!-padding-bottom-2 print-only'
+                        },
+                        {
+                            html: '<span class="print-checkbox">S</span>',
+                            classes: 'govuk-table__cell govuk-!-padding-top-2 govuk-!-padding-bottom-2 print-only'
+                        },
+                        {
+                            html: '<span class="print-checkbox">G</span>',
+                            classes: 'govuk-table__cell govuk-!-padding-top-2 govuk-!-padding-bottom-2 print-only'
+                        },
+                        {
+                            classes: 'govuk-table__cell govuk-!-padding-top-2 govuk-!-padding-bottom-2 print-only',
+                            html: ''
+                        }
+                    ]), rows) %}
+                {% endfor %}
+                {{ govukTable({
+                    attributes: {
+                        'data-module': 'moj-sortable-table',
+                        id: 'unlock-list-table'
+                    },
+                    caption: "Prisoners",
+                    classes: "alternating-row-shading fixed-layout-table",
+                    captionClasses: "govuk-visually-hidden",
+                    head: [
+                        {
+                            text: "Name",
+                            attributes: { "aria-sort": "ascending" },
+                            classes: 'govuk-table__header'
+                        },
+                        {
+                            text: "Cell location",
+                            attributes: { "aria-sort": "ascending" },
+                            classes: 'govuk-table__header'
+                        },
+                        {
+                            html: "<span>Events</span>",
+                            attributes: { "aria-sort": "none" },
+                            classes: 'govuk-table__header'
+                        },
+                        {
+                            html: "<span>Refused</span>",
+                            classes: 'govuk-table__header hmpps-table__header--angled-table-header govuk-!-padding-right-1 print-only'
+                        },
+                        {
+                            html: "<span>Sick</span>",
```

Comment:
The span shouldn't have been included on the events column – I've removed that.

The span wraps the text itself so the text can be rotated directly without rotating the table header. Rotating the th would move the text relative to the rotation of the th, this would mean accounting for things like cell padding in the rotation. I felt this solution is slightly more elegant as you don't need to worry about how the text moves relative to the th.

I also thought it looked neater when inspecting the html:

**span rotation**
<img width="394" alt="Screenshot at Feb 20 12-11-57" src="https://user-images.githubusercontent.com/125488090/220103294-0fb5ceb1-6f58-460f-8831-742eb1242b23.png">

**th rotation**
<img width="393" alt="Screenshot at Feb 20 12-16-12" src="https://user-images.githubusercontent.com/125488090/220104491-cc7f6b7d-fc1b-4840-9b9c-002d36dc70b5.png">